src/main/java/tectech/thing/metaTileEntity/multi/MTEEnergyInfuser.java

```diff
@@ -62,12 +66,12 @@ public class MTEEnergyInfuser extends TTMultiblockBase implements ISurvivalConst
         .addElement('B', ofBlock(TTCasingsContainer.sBlockCasingsTT, 7))
         .addElement(
             'C',
-            ofHatchAdderOptional(
-                MTEEnergyInfuser::addClassicToMachineList,
-                BlockGTCasingsTT.textureOffset,
-                1,
-                TTCasingsContainer.sBlockCasingsTT,
-                0))
+            buildHatchAdder(MTEEnergyInfuser.class)
+                .atLeast(Energy.or(HatchElement.EnergyMulti), Maintenance, InputBus, OutputBus)
+                .casingIndex(BlockGTCasingsTT.textureOffset) // High Power Casing
+                .dot(1)
+                .buildAndChain(TTCasingsContainer.sBlockCasingsTT, 0)) // High Power Casing
```

Comment:
I'd like to see this swapped to use Casings.java instead, but I won't nitpick since this PR is so old.

```suggestion
                .casingIndex(Casings.HighPowerCasing.getTextureId())
                .dot(1)
                .buildAndChain(Casings.HighPowerCasing.asElement())
```

Reply:
if it works, sure